p4include/v1model.p4

```diff
@@ -104,7 +104,7 @@ extern action_profile {
 }
 
 // Get a random number in the range lo..hi
-extern void random(out bit<32> result, in bit<32> lo, in bit<32> hi);
+extern void random<T>(out T result, in T lo, in T hi);
```

Comment:
This is more convenient for software switches, but is it good for hardware switches?

Reply:
Hardware switches may require that lo == 0 and hi == one less than a power of 2, but that's not really affected by this change.